pkg/types/stream.go

```diff
@@ -129,10 +134,12 @@ type StandardStreamEmitter interface {
 }
 
 func NewStandardStream() StandardStream {
-	return StandardStream{
+	s := StandardStream{
 		ReconnectC: make(chan struct{}, 1),
 		CloseC:     make(chan struct{}),
 	}
+	s.heartBeat = s.ping(pingInterval)
```

Comment:
I think we still need the under layer ping/pong worker, you should not replace it